legion/legion/edi/security.py

```diff
@@ -134,3 +134,14 @@ def get_security_params_from_config():
             LOG.debug('Exception during parsing of legion config {}'.format(e), exc_info=True)
 
     return {}
+
```

Comment:
Also, please replace the http request by legionctl command:
* https://github.com/legion-platform/legion/blob/develop/tests/robot/resources/keywords.robot#L182